s3/sc-s3-encrypted-ra.yaml

```diff
@@ -51,6 +51,14 @@ Resources:
         ServerSideEncryptionConfiguration:
           - ServerSideEncryptionByDefault:
               SSEAlgorithm: AES256
+      LifecycleConfiguration:
+        Rules:
+          - Id: IntelligentTieringRule
```

Comment:
I would call this IntelligentTieringClassTransitionRule

Reply:
done